src/code/z_demo.c

```diff
@@ -24,31 +51,59 @@ CutsceneStateHandler sCsStateHandlers2[] = {
 
 u8 sTitleCsState = 0;
 
-// TODO: use symbols for these cutscene locations once scenes and rooms are in C
+// Todo use enum when one is created
+#define AGE_REQ_CHILD 1
+#define AGE_REQ_ADULT 0
+#define AGE_REQ_NONE 2
```

Comment:
Imo this doesn't really belong in this PR or at least warrants more discussion.

I personally don't think we should use any special defines here since the values actually correspond to `linkAge`, except `2` which is used as an arbitrary value and might warrant a define but I'm not sure it's even worth one tbh